Backend/PO.Project.Domain/Service.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace PO.Project.Domain
+{
+    public class Service
+    {
+        public int IdService { get; set; }
+        public string ServiceName { get; set; }
+        public double Price { get; set; }
+        public string ApproxTime { get; set; }
+
+        public int IdRepair { get; set; }
+        public Repair Repair { get; set; }
```

Comment:
Wszędzie powinno być virtual:
https://stackoverflow.com/questions/8542864/why-use-virtual-for-class-properties-in-entity-framework-model-definitions